app/controllers/application_controller.rb

```diff
@@ -207,7 +207,7 @@
     return if request.host.match?(/(^|\.)glowfic\.com$/)
     return if request.host.match?(/(^|\.)glowfic-staging\.herokuapp\.com$/)
     glowfic_url = root_url(host: ENV.fetch('DOMAIN_NAME'), protocol: 'https')[0...-1] + request.fullpath # strip double slash
-    redirect_to glowfic_url, status: :moved_permanently
+    redirect_to glowfic_url, status: :moved_permanently, allow_other_host: true
```

Comment:
Not worse than the previous configuration. _Should_ overall be safe(?) because we specify the host, but maybe there's some way to escape our domain with a carefully crafted path?

Seems OK to ignore IMO.